api/src/test/java/io/grpc/ForwardingTestUtil.java

```diff
@@ -97,7 +97,7 @@ public Object get(Method method, int argPos, Class<?> clazz) {
       try {
         method.invoke(verify(mockDelegate), args);
       } catch (InvocationTargetException e) {
-        throw new AssertionError(String.format("Method was not forwarded: %s", method));
+        throw new RuntimeException(String.format("Method was not forwarded: %s", method), e);
```

Comment:
There's no need to subclass AssertionError. If we can't use the two-arg constructor because of our Android restrictions, then:

```java
throw new AssertionError(String.format("Method was not forwarded: %s", method)).initCause(e);
```